lib/ansible/parsing/vault/__init__.py

```diff
@@ -344,23 +343,21 @@ def __repr__(self):
 
 
 class ScriptVaultSecret(FileVaultSecret):
-
-    @staticmethod
-    def read_file(filename, loader):
-        if not loader.is_executable(filename):
+    def read_file(self, filename):
+        if not self.loader.is_executable(filename):
             raise AnsibleVaultError("The vault password script %s was not executable" % filename)
 
         try:
             # STDERR not captured to make it easier for users to prompt for input in their scripts
             p = subprocess.Popen(filename, stdout=subprocess.PIPE)
         except OSError as e:
-            msg_format = "Problem running vault password script %s (%s)."
-            "If this is not a script, remove the executable bit from the file."
-            msg = msg_format % (' '.join(filename), e)
+            msg_format = "Problem running vault password script %s (%s)." \
+                " If this is not a script, remove the executable bit from the file."
+            msg = msg_format % (filename, e)
 
             raise AnsibleError(msg)
 
-        stdout, stderr = p.communicate()
+        stdout, _ = p.communicate()
```

Comment:
```
newswoop:F25:ansible (vault_pylint %)]$ ack --python ' _+ ' | wc -l
82
# the (_[a-zA-Z0-9_]*[a-zA-Z0-9]+?$)  regex hits _anything, private methods etc ~7000 hits, afaict none of them to indicate unused vars
# ignored doesnt seem to be used much to indicate unused vars but is in a lot of var names
# unused_ gets no matches.
```

So of the defaults, _ is the only one we use, and we've used it 82 times, most to indicate unused/dummy vars (including a view usages of 'for _ in range(some_count)'.